src/containers/cityDetails.container.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { cityActions ,forecastActions} from '../actions';
+import { connect } from 'react-redux';
+
+class CityDetailsContainer extends React.Component {
+    componentDidMount(){
+        const {getCity,getForecast} = this.props;
+        getForecast(getCity(this.props.match.params.place_id));
```

Comment:
I don't think you need 2 actions here. Probably it should be only 1 and all logic on how to load data from different services (city service and forecast service) should be out of redux actions. For example here  `src/services`